rfc/custom-metric-modules.md

```diff
@@ -0,0 +1,196 @@
+---
```

Comment:
Is this RFC still valid? You already addressing custom metrics support in your other [RFC](https://github.com/wundergraph/cosmo/pull/1130/files#diff-08e3487dd24ec2c179de58a713f2627b95d28796cffd282c9af47d151946ce9eR365).

Reply:
Let me close this one and we'll just use the other RFC.